settings.gradle.kts

```diff
@@ -33,7 +33,7 @@ dependencyResolutionManagement {
     mavenLocal()
 
     maven {
-      setUrl("https://oss.sonatype.org/content/repositories/snapshots")
+      setUrl("https://central.sonatype.com/repository/maven-snapshots/")
```

Comment:
Probably include it in the final PR: https://github.com/aws-observability/aws-otel-java-instrumentation/pull/1090/files